common/config/rush/pnpm-lock.yaml

```diff
@@ -1,5 +1,10 @@
 lockfileVersion: 5.3
 
+overrides:
```

Comment:
Where are these comming from ? Were they added manually ? Will they get wiped next time we `rush update` ?

Reply:
Was a first attempt. They should be gone on update, I can do that and test everything works.

Reply:
Gone.

Reply:
Was an issue with the validation in core... should hopefully be fixed now... let's see